recipe/meta.yaml

```diff
@@ -18,11 +18,11 @@ outputs:
   - name: {{ name }}
     build:
       script: python -m pip install --no-deps --ignore-installed .
-      # noarch: python
 
     requirements:
       host:
         - python
+        - flit-core >=3.4,<4
```

Comment:
do we want to specify here any flit-core version available in this range for building? or do we intend to specify minimum version here?  As by checking available version in that range for all archs, there is just 3.6.0 ...
so I would suggest to add to host '- flit-core 3.6.0'
and then add to run-section '- flit-core >=3.6,<4'

Reply:
This has been updated.